utm.yaml

```diff
@@ -1317,6 +1271,213 @@ components:
         status:
           $ref: '#/components/schemas/UssAvailabilityStatus'
 
+    RequestResponse:
+      type: object
+      description: Details of a request/response data exchange.
+      required:
+      - url
+      - method
+      - request_time
+      - request_body
```

Comment:
If the request body is empty, we should be able to omit this field with a default value of `''`.  Same comment for `response_body`.